compliance-web/src/utils/axiosUtils.ts

```diff
@@ -1,42 +1,42 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { AppConfig, OidcConfig } from '@/utils/config'
-import axios, { AxiosError } from 'axios'
-import { User } from "oidc-client-ts"
+import { AppConfig, OidcConfig } from "@/utils/config";
+import axios, { AxiosError } from "axios";
+import { User } from "oidc-client-ts";
 
 export type OnErrorType = (error: AxiosError) => void;
 export type OnSuccessType = (data: any) => void;
 
 const client = axios.create({ baseURL: AppConfig.apiUrl });
```

Comment:
Shall we put a 3 times retry option with axios.

Reply:
Tanstack query is already doing that by default, I don't think we need to do it in Axios. we got more controls via TSQ